File: scenes/menu/Menu.cs
using Godot;

public class Menu : Base
{
    public void OnStartPressed()
    {
        SwitchTo("res://scenes/game/Diner.tscn");
    }

    public void OnAboutPressed()
    {
        SwitchTo("res://scenes/about/About.tscn");
    }
}

File: scenes/menu/Menu.fs
namespace BloodFeverFs.Scene

type Menu() =
    inherit Base()

    member this.OnStartPressed() =
        this.SwitchTo("res://scenes/game/Diner.tscn")

    member this.OnAboutPressed() =
        this.SwitchTo("res://scenes/about/About.tscn")
